add2cal/add2cal.py

```diff
@@ -29,14 +29,17 @@ def __init__(
         end,
         description,
         location,
-        timezone
+        timezone,
+        attendee
```

Comment:
tests in this repo need to be updated to pass in the now required 'attendee' field. currently they are failing with: 
```TypeError: __init__() missing 1 required positional argument: 'attendee'```